views/elements/current_and_next.tmpl

```diff
@@ -25,7 +25,7 @@
     {{else}}
     </div>
     {{end}}
-    <a href="https://ury.org.uk/listen" class="btn btn-red listen-btn-animated mobile-hide bounceInUp" title="Listen Live" onclick="window.open('http://ury.org.uk/live', 'radioplayer', 'height=665,width=380'); return false;"><i class="fa fa-volume-up mr-2"></i>Listen Live</a>
+    <a href="https://ury.org.uk/listen" class="btn btn-red listen-btn listen-btn-animated mobile-hide" title="Listen Live" onclick="window.open('http://ury.org.uk/live', 'radioplayer', 'height=665,width=380'); return false;"><i class="fa fa-volume-up mr-2"></i>Listen Live</a>
```

Comment:
`window.open('http://....` doesn't match the href and isn't https...

Reply:
I believe that this was initially intentional, due to radio player not working with JS disabled, where as the listen site does work with JS disabled. Should certainly be https though :)